src/Symfony/Component/Security/Core/Resources/translations/security.ar.xlf

```diff
@@ -66,6 +66,15 @@
                 <source>Account is locked.</source>
                 <target>الحساب مغلق.</target>
             </trans-unit>
+            <trans-unit id="17">
+                <source>Too many failed login attempts, please try again later.</source>
+                <target>عدد كبير جدا من محاولات الدخول الفاشلة، يرجى المحاولة مرة أخرى في وقت لاحق.
+</target>
```

Comment:
Extra newline apparently.

Reply:
you are right, fixed in the last commit :)